dbms/src/Storages/DeltaMerge/StableValueSpace.h

```diff
@@ -141,14 +157,14 @@ class StableValueSpace : public std::enable_shared_from_this<StableValueSpace>
 
         ColumnCachePtrs column_caches;
 
-        Snapshot()
-            : log(&Poco::Logger::get("StableValueSpace::Snapshot"))
+        Snapshot(StableValueSpacePtr stable_)
+            : stable(stable_)
+            , log(Logger::get("StableValueSpace::Snapshot", fmt::format("<{}>", stable->log_prefix)))
```

Comment:
Maybe use `stable->log->identifier()` instead of saving `log_prefix` in `StableValueSpace`?

Reply:
Updated. Now there are still some `log_prefix` variables remain, when we need to concat multiple properties together. We may resolve it in https://github.com/pingcap/tiflash/issues/6068